inc/class-ac-yoast-acf-content-analysis.php

```diff
@@ -223,4 +227,20 @@ protected function get_blacklist_type() {
 	protected function get_blacklist_name() {
 		return new Yoast_ACF_Analysis_String_Store();
 	}
+
+	/**
+	 * Adds a CPT for tests
```

Comment:
Can you make this line more clear. I know where CPT stands for, but somebody who's new to the codebase probably won't know.